components/builder-sessionsrv/src/server/mod.rs

```diff
@@ -64,30 +66,51 @@ lazy_static! {
     };
 }
 
-#[derive(Clone)]
+pub type SessionKey = (u32, proto::OAuthProvider);
+
+#[derive(Clone, Debug)]
 pub struct Session {
     pub created_at: Instant,
+    pub key: SessionKey,
     inner: proto::Session,
+    token: proto::SessionToken,
```

Comment:
We're going to need to get some documentation strings in here (i.e., the codebase, not just this particular site) soon so we can keep straight exactly what each component is.

Reply:
yeah.. personally I'd prefer cleaning up naming where needed vs. more documentation...

Reply:
Yeah, either way is fine; I just want to reduce the "bus number" for this code soon 😄

Reply:
Yeah I want to clean up the nomenclature here just as soon as we stabilize sessions but the gist is this:

* Session - contains all information you need to provide base operability upon login. It should include things like the origin's you're part of, your name, your id, and your token
* SessionToken - opaque piece of identifiable data which is transient and lasts for 1 day. It's packed with information for our server to identify you upon a login. You should never give somebody your session token because they can _become you_